lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -224,7 +222,7 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
       data-display-name="${_("Discussion Admins")}"
       data-info-text="
         ${_("Discussion Admins can edit or delete any post, clear misuse flags, close "
-         "and re-open threads, endorse responses, and see posts from all cohorts. "
+         "and re-open threads, endorse responses, and see posts from all groups. "
```

Comment:
"You can only give course team roles to enrolled users." -> "Only enrolled users can be added as Discussion Admins."